src/appState.ts

```diff
@@ -74,6 +74,12 @@ export const getDefaultAppState = (): Omit<
     zenModeEnabled: false,
     zoom: { value: 1 as NormalizedZoomValue, translation: { x: 0, y: 0 } },
     viewModeEnabled: false,
+    canvasActions: {
```

Comment:
why do we need a state for this ? We can use the prop itself to decide whether to show or not.